### PR TITLE
Tobin gpdp2 GitHub merge

### PR DESCRIPTION
**Pull request by tobine**
Port validation for get_physical_device_properties2 extension over to GitHub now that it's public.

### CODE REVIEW

File: layers/parameter_validation.cpp [hunk 1]
```diff
@@ -5659,57 +5676,80 @@ VKAPI_ATTR VkResult VKAPI_CALL GetPhysicalDeviceImageFormatProperties2KHR(
...
+    if (!skip) {
         my_data->dispatch_table.GetPhysicalDeviceMemoryProperties2KHR(physicalDevice, pMemoryProperties);
     }
 }
 
+static bool PostGetPhysicalDeviceSparseImageFormatProperties2KHR(VkPhysicalDevice physicalDevice,
+                                                                 const VkPhysicalDeviceSparseImageFormatInfo2KHR *pFormatInfo,
+                                                                 uint32_t *pPropertyCount,
+                                                                 VkSparseImageFormatProperties2KHR *pProperties) {
+    auto my_data = get_my_data_ptr(get_dispatch_key(physicalDevice), instance_layer_data_map);
```
Comment (mark-lunarg) [inline layers/parameter_validation.cpp]: Nitpick -- it's present throughout the layers, but can we use anything besides 'my_' for the layer data?

**Comment by mark-lunarg** [comment]
Also, for some reason the appveyor build choked on 'qfp'.  Might be spurious.

File: layers/parameter_validation.cpp (hunk 1 above)
Comment (tobine) [inline layers/parameter_validation.cpp]: For sure. I copied these changes directly from GitLab, but after this lands I'll do a wholesale update to core_validation to kill all of the "my_" vars.

Comment (mark-lunarg) [inline layers/parameter_validation.cpp]: hallelujah 🥇

**Pull request merged**